pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -218,7 +218,9 @@ def restore_loops(self) -> None:
             return
 
         self.trainer.fit_loop.global_step = self._loaded_checkpoint["global_step"]
```

Comment:
are you planning to track global_step in the progress too in the future? since this is being updated manually everywhere.

Reply:
Yes!

Reply:
We can discuss in #7406